web/packages/plugins/analysis-platform/src/modules/Analysis/OperatorView.tsx

```diff
@@ -15,6 +15,7 @@ export const OperatorView = (props: DmtUIPlugin): JSX.Element => {
   const [jobs, setJobs] = useState<any[]>([])
   const [analysis, setAnalysis] = useState<any>()
   const [document, loading] = useDocument(dataSourceId, documentId)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore